docs/tutorials/google/visualizing_calibration_metrics.ipynb

```diff
@@ -0,0 +1,264 @@
+{
```

Comment:
<p>theoretically we have the project id now from the previous cell, so you can reuse that here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/3880/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
done